avr-specs/avr-attiny841.json

```diff
@@ -0,0 +1,32 @@
+{
+  "arch": "avr",
+  "atomic-cas": false,
+  "cpu": "avr25",
+  "data-layout": "e-P1-p:16:8-i8:8-i16:8-i32:8-i64:8-f32:8-f64:8-n8-a:8",
+  "eh-frame-header": false,
+  "env": "",
+  "exe-suffix": ".elf",
+  "executables": true,
+  "late-link-args": {
+    "gcc": [
+      "-lgcc"
+    ]
+  },
+  "linker": "avr-gcc",
+  "linker-flavor": "gcc",
+  "linker-is-gnu": true,
+  "llvm-target": "avr-unknown-unknown",
+  "max-atomic-width": 8,
+  "no-default-libraries": false,
+  "os": "unknown",
+  "pre-link-args": {
+    "gcc": [
+      "-mmcu=attiny841",
+      "-Wl,--as-needed"
+    ]
+  },
+  "target-c-int-width": "16",
+  "target-endian": "little",
+  "target-pointer-width": "16",
+  "vendor": "unknown"
+}
```

Comment:
The target specs should be generated via [`sync-from-upstream.py`](https://github.com/Rahix/avr-hal/blob/master/avr-specs/sync-from-upstream.py), can you integrate your two MCUs into that script?